apps/ui/src/composables/useVoteVotingPower.ts

```diff
@@ -0,0 +1,65 @@
+import { supportsNullCurrent } from '@/networks';
+import { getIndex } from '@/stores/votingPowers';
+import { NetworkID, Proposal, Space } from '@/types';
+
+export function useVoteVotingPower() {
```

Comment:
I think we could also make the names more clear now to avoid using vote voting power and proposal voting power, in contracts we started using following names:
- voting power
- proposition power

Could be nice to introduce it here as well to avoid this confusing naming.

Reply:
This should be done after we also refactor to have a dedicated composable for proposition power, else composable name will conflict with existing `useVotingPower`

Reply:
I was thinking, we could call this `usePropositionPower` already (going in the right direction in terms of naming), with `useVoteVotingPower` we are introducing new name that would be removed in the future.

Reply:
This PR is refactoring the voting power, and not the proposition power.

To avoid naming collision, we could start the refactoring on the proposition power, and refactor this one with the new name AFTER.

Reply:
We should also decide on a naming convention on the exported function (`getProposalVp`, `getSpaceVp` ...) since VP can not be used with proposition power.

Should we go with `getSpaceVp` and `getSpacePp` (get space proposition power) ?

Reply:
Let's try to merge https://github.com/snapshot-labs/sx-monorepo/pull/988 first, to avoid file name collision, so this PR can be refactored to used the final name `useVotingPower`.